README.md

```diff
@@ -17,7 +35,7 @@ For Ansible, several roles are required, and you can install them as follows:
 
     ansible-galaxy install -r requirements.yml
```

Comment:
It is just a minor, but "ansible-galaxy" is also required in case of vagrant, so could you move "ansible-galaxy" into non-openstack section in this README?